### PR TITLE
If QuotaControl service is not available, return utils::Status::OK

### PR DESCRIPTION
**Pull request by mangchiandjjoe**
@qiwzhang

### CODE REVIEW

File: contrib/endpoints/src/api_manager/quota_control.cc [hunk 1]
```diff
@@ -46,7 +46,10 @@ void QuotaControl(std::shared_ptr<context::RequestContext> context,
         TRACE(trace_span) << "Quota service control request returned with "
                           << "status " << status.ToString();
 
-        continuation(status);
+        // if the quota control service API is not available, then pass client
+        // requests
+        continuation((status.code() == Code::UNAVAILABLE) ? utils::Status::OK
```
Comment (qiwzhang) [inline contrib/endpoints/src/api_manager/quota_control.cc]: Should we do this in ConvertQuotaResponse() function in proto.cc?

Comment (mangchiandjjoe) [inline contrib/endpoints/src/api_manager/quota_control.cc]: In Aggregated::Quota, we call Proto::ConvertAllocateQuotaResponse when the response of the Call is OK.

This is to handle Code::UNAVAILABLE from Aggregated::Call

Comment (mangchiandjjoe) [inline contrib/endpoints/src/api_manager/quota_control.cc]: Aggregated::Call returns Code::UNAVAILABLE when the backend api returns non 200 OK response.

File: contrib/endpoints/src/api_manager/quota_control.cc [hunk 2]
```diff
@@ -46,7 +46,10 @@ void QuotaControl(std::shared_ptr<context::RequestContext> context,
         TRACE(trace_span) << "Quota service control request returned with "
                           << "status " << status.ToString();
 
-        continuation(status);
+        // if the quota control service API is not available, then pass client
+        // requests
```
Comment (qiwzhang) [inline contrib/endpoints/src/api_manager/quota_control.cc]: I see. could you update the comment
// quota control is using "failed open" policy.  If the server is not available, allow the request to go.

**Pull request merged**